lib/tests/unit/transforms/test_core.py

```diff
@@ -161,6 +161,23 @@ def test_load_config_from_file_raises(config_dict, mocker, monkeypatch):
         core_transforms.RunConfig._load_config_from_file()
 
 
+# FIXME: other test files (test_pubsub_message_manager.py and others) patch
```

Comment:
Just confirming: this is because our tests run in parallel and even resetting the value to None won't guarantee it will be empty when we call `.get()` if another test loads the config first?

Reply:
@shireen-bean I _suspect_ (@DanSimon can correct me if I'm wrong) that this is because patches are "leaking" from other tests. FWIW our tests are not run in parallel.

Reply:
I think I've gotten around the issue by adding some `patcher.stop()` calls.  I've updated the PR with my changes, tests all pass so it definitely works, though I'm open if there's another approach we should take.